pkgs/by-name/ta/tailscale-nginx-auth/package.nix

```diff
@@ -0,0 +1,45 @@
+{ lib, stdenv, buildGoModule, fetchFromGitHub, makeWrapper }:
```

Comment:
makewrapper isn't needed for nginx-auth, you can remove it (it's used to put network tools in the $PATH of tailscaled)